### ISSUE TITLE
Setting the pose of the models through code

### ISSUE THREAD

**Issue by kevindokaist** [osrf/mbzirc#174]
Hi, we want to test out our method for various problem configurations such as different initial locations of the vessels.

In Gazebo, there is a topic or service `/gazebo/set_model_state` that is subscribed by Gazebo and change the states of the models.

Is there any way to do this in the provided simulator?

Thank you!

**Comment by iche033** [comment]
yeah we have a service for teleporting entities. Assuming your USV is named `usv`, you can move it like so:

```
ign service -s /world/coast/set_pose --reqtype ignition.msgs.Pose --reptype ignition.msgs.Boolean --timeout 300 --req 'name: "usv", position: {x: -1400, y: 1, z: 0.3}'
```
